app/src/main/kotlin/com/wire/android/ui/home/gallery/MediaGalleryViewModel.kt

```diff
@@ -107,15 +108,19 @@ class MediaGalleryViewModel @Inject constructor(
 
     fun deleteCurrentImageMessage() {
         viewModelScope.launch {
+            navigationManager.navigateBack
```

Comment:
Is this necessary? 🤔

Reply:
omg, it may have slipped through all the testing I was doing 😮‍💨